internal/controller/runtime/runtime_controller.go

```diff
@@ -57,10 +57,21 @@ func (r *RuntimeReconciler) Reconcile(ctx context.Context, request ctrl.Request)
 		}, client.IgnoreNotFound(err)
 	}
 
-	r.Log.Info("Reconciling Runtime", "Name", runtime.Name, "Namespace", runtime.Namespace)
+	runtimeID, ok := runtime.Labels["kyma-project.io/runtime-id"]
+	if !ok {
+		runtimeID = runtime.Name
+	}
+
+	shootName, ok := runtime.Labels["kyma-project.io/shoot-name"]
+	if !ok {
+		shootName = "N/D"
+	}
+
+	log := r.Log.WithValues("runtime", runtimeID, "shoot", shootName)
+	log.Info("Reconciling Runtime", "Name", runtime.Name, "Namespace", runtime.Namespace)
```

Comment:
The overall idea is nice, and this is the correct place to implement that change. I tested it on local environment at it looks good, but I have one minor remark:
```
2024-11-22T12:03:04+01:00       INFO    Reconciling Runtime     {"runtime": "aaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa", "shoot": "test-load", "Name": "test-ci-m5t28", "Namespace": "kcp-system"}
```
```
2024-11-22T12:03:04+01:00       INFO    reqID 0 reconciliation done     {"runtime": "aaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa", "shoot": "test-load", "error": null, "result": null}
```
Could you change in line 70 `runtime` to `runtimeID` to indicate that log is printing runtimeID?